libirods_rule_engine_plugin-elasticsearch.cpp

```diff
@@ -333,27 +337,42 @@ namespace
 				// The data to index.
 				// The version of .dump() invoked here instructs the library to ignore
 				// invalid UTF-8 sequences. All bytes are copied to the output unchanged.
+				// clang-format off
 				ss << json{
 					{"absolutePath", _object_path},
 					{"data", std::string_view(buffer.data(), in.gcount())}
 				}.dump(indent, indent_char, ensure_ascii, json::error_handler_t::ignore) << '\n';
+				// clang-format on
 
 				// Send bulk request if chunk counter has reached bulk limit.
 				if (chunk_counter == config->bulk_count_) {
 					chunk_counter = 0;
-					const auto res = send_http_request(config->hosts_[0], http::verb::post, _index_name + "/_bulk", ss.str()); // TODO C++20 supports .view(), but clang 13 doesn't appear to implement it :-(
-					(void) res;
-					// TODO Check response.
+
+					const auto res = send_http_request(config->hosts_[0], http::verb::post, _index_name + "/_bulk", ss.str());
+
+					if (!res.has_value()) {
+						rodsLog(LOG_ERROR, "%s: No response from elasticsearch host.", __func__);
```

Comment:
Should this be `LOG_ERROR`?

Reply:
It was changed to LOG_ERROR. Seems I changed it before your review was posted.